lib/ioki/webhooks/signature_validator.rb

```diff
@@ -18,25 +18,28 @@
 module Ioki
   module Webhooks
     class SignatureValidator
-      def initialize(body:, signature:)
+      def initialize(body:, signature:, signature_key:)
```

Comment:
This is a breaking change. We don't have a proper release process yet to highlight this (although we do have a version file, so we just at least bump that).

However, I wonder if we should go for this for now:

```suggestion
      def initialize(body:, signature:, signature_key: ENV.fetch('WEBHOOK_SIGNATURE_KEY', nil))
```

This would allow dependent projects to do the upgrade and once we have a release process in place, we could easily increase the major version with proper release notes.

Or do you reckon it's fine as is, as this would throw an ArgumentError if you just update the dependency without doing the necessary call update and rely on the respective test suites to catch that?

Reply:
Fair point! Indeed it would throw an error and we should mention it in the release notes. But I like your idea of providing  the ENV variable by default, it is the most seamless transition.